src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -207,6 +211,29 @@ private void incrementTrait(String key, Object value, Identify identify) {
     }
   }
 
+  private void setOnce(String key, Object value, Identify identify) {
```

Comment:
We should have a test for all of these cases - when a trait is double, float, integer and long.